yarn-project/sequencer-client/src/tx_validator/gas_validator.ts

```diff
@@ -65,10 +66,26 @@ export class GasTxValidator implements TxValidator<Tx> {
     const feeLimit = tx.data.constants.txContext.gasSettings.getFeeLimit();
 
     // Read current balance of the feePayer
-    const initialBalance = await this.#publicDataSource.storageRead(
+    // TODO(#11285): Remove the 2 reads below with the commented out code.
+    // Uncomment below ######################
+    // const initialBalance = await this.#publicDataSource.storageRead(
+    //   this.#feeJuiceAddress,
+    //   computeFeePayerBalanceStorageSlot(feePayer),
+    // );
+    // Uncomment above ######################
+    // Remove the following ######################
+    const initialBalanceLowLimb = await this.#publicDataSource.storageRead(
       this.#feeJuiceAddress,
       computeFeePayerBalanceStorageSlot(feePayer),
     );
+    const initialBalanceHighLimb = await this.#publicDataSource.storageRead(
+      this.#feeJuiceAddress,
+      new Fr(computeFeePayerBalanceStorageSlot(feePayer).toBigInt() + 1n),
+    );
+    const initialBalance = new Fr(
+      U128.fromU64sLE(initialBalanceLowLimb.toBigInt(), initialBalanceHighLimb.toBigInt()).toInteger(),
+    );
+    // Remove the above ######################
```

Comment:
U128 now occupies 2 storage slots so I had to make 2 storage reads here. This will change back to one read once I implement Packable trait and use it for contract storage.